src/main/java/roomescape/reservation/ReservationService.java

```diff
@@ -0,0 +1,123 @@
+package roomescape.reservation;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import roomescape.reservationTheme.ReservationTheme;
+import roomescape.reservationTheme.ReservationThemeResponseDto;
+import roomescape.reservationTime.ReservationTime;
+import roomescape.reservationTime.ReservationTimePolicy;
+import roomescape.reservationTime.ReservationTimeResponseDto;
+
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class ReservationService {
+
+    private final ReservationRepository reservationRepository;
+
+    public ReservationService(ReservationRepository reservationRepository) {
+        this.reservationRepository = reservationRepository;
+    }
+
+    public List<ReservationResponseDto> findAll() {
+        List<Reservation> reservations = reservationRepository.findAll();
+        return reservations.stream()
+                .map(
+                        reservation -> new ReservationResponseDto.Builder()
+                                .id(reservation.getId())
+                                .name(reservation.getName())
+                                .date(reservation.getDate())
+                                .reservationTimeResponseDto(new ReservationTimeResponseDto(
+                                        reservation.getReservationTime().getId(),
+                                        reservation.getReservationTime().getStartAt()))
+                                .reservationThemeResponseDto(new ReservationThemeResponseDto(
+                                        reservation.getReservationTheme().getId(),
+                                        reservation.getReservationTheme().getName(),
+                                        reservation.getReservationTheme().getDescription(),
+                                        reservation.getReservationTheme().getThumbnail()
+                                ))
+                                .build()
+                ).toList();
```

Comment:
```suggestion
        return reservations.stream()
            .map(reservation -> new ReservationResponseDto.Builder()
                .id(reservation.getId())
                .name(reservation.getName())
                .date(reservation.getDate())
                .reservationTimeResponseDto(new ReservationTimeResponseDto(
                    reservation.getReservationTime().getId(),
                    reservation.getReservationTime().getStartAt()))
                .reservationThemeResponseDto(new ReservationThemeResponseDto(
                    reservation.getReservationTheme().getId(),
                    reservation.getReservationTheme().getName(),
                    reservation.getReservationTheme().getDescription(),
                    reservation.getReservationTheme().getThumbnail()
                )).build())
            .toList();
```
엄청 기술적인 내용은 아니지만, 의외로 공백을 관리하는게 가독성에 영향을 많이 준다고 생각해요.
지금 그렇지는 않지만 나중에 로직이 더 복잡하거나 변수 이름들이 길어졌을 때도 영향을 주기도하고요.

(+) 서비스 레이어는 변환 로직을 포함하지 않아도 시간이 갈수록 굉장히 비대해지는 공간이에요.
그러니 이 변환 작업을 다른 곳으로 옮겨보는건 어떨까요?
예를 들어 DTO 에 정적 팩토리 메서드를 만들어볼 수 있겠네요 :)